### ISSUE TITLE
[Accessibility][Keyboard navigation] 'Remove' button does not appear in 'Suggested contact list' window on keyboard focus and is also not accessible. [Duplicated by #4816]

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11022]
Breakout of #6259

**Comment by joschect** [comment]
This is no longer an issue, the "delete" button correctly removes the item from the list.
